spec/SideBar.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow } from 'enzyme';
+
+import SideBar from '../client/src/SideBar.jsx';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
If you end up with a lot of components, you can create a `setupTests.js` file and configure jest to load it before each test so you don't have to repeat the Enzyme.configure in every file.  With a few components, it's not really a big deal though:
https://github.com/airbnb/enzyme/blob/master/docs/guides/jest.md

Reply:
Got it!